cunumeric/sort.py

```diff
@@ -79,7 +79,7 @@ def sort_task(output, input, argsort, stable):
         task.add_broadcast(input.base)
 
     task.add_scalar_arg(argsort, bool)  # return indices flag
-    task.add_scalar_arg(input.base.shape, (ty.int32,))
+    task.add_scalar_arg(input.base.shape, (ty.int64,))
```

Comment:
Make sure you merge this with the latest branch-22.05. this looks like a change you already made in the other pr.